starter_code/src/movies.js

```diff
@@ -4,8 +4,32 @@
 
 // Get the average of all rates with 2 decimals 
 
+function reduceFunction (sum, item){
+    return sum + item.rate;
+}
+
+function ratesAverage(arr) {
+    var averageRate = arr.reduce(reduceFunction , 0) / arr.length;
+    return averageRate;
+}
+
+//  Get the average of Drama Movies
+
+ function dramaMoviesRate(item){
+     return item.genre.includes("Drama");
+ }
+
+ function reduceDrama (sum, item, index){
```

Comment:
Don't you feel like you are repeating yourselves? You can reuse the function that you write before (reduceFunction) to calculate the sum.